src/ucp/wireup/wireup_ep.c

```diff
@@ -52,11 +52,31 @@ static ssize_t ucp_wireup_ep_bcopy_send_func(uct_ep_h uct_ep)
     return UCS_ERR_NO_RESOURCE;
 }
 
+static int ucp_wireup_ep_is_next_ep_active(ucp_wireup_ep_t *wireup_ep)
+{
+    return (wireup_ep->flags & UCP_WIREUP_EP_FLAG_READY) ||
+           (wireup_ep->aux_ep == NULL);
+}
+
+uct_ep_h ucp_wireup_ep_extract_msg_ep(ucp_wireup_ep_t *wireup_ep)
```

Comment:
`extract` is unclear.
How is it different from `get`? What does it add?

Reply:
on the same weight as `ucp_wireup_ep_extract_next_ep`